src/routes.tsx

```diff
@@ -154,17 +154,28 @@ export const reactPageRoutes: ReactPageRoute[] = [
     path: LOCALE_PATH_PATTERN + '/new-member/:name?/:id?',
     render: ({ match }: RouteComponentProps<any>) => {
       const getProps = () => {
+        match.params
         switch (match.params.locale) {
           case 'dk':
-            return {
-              baseUrl: '/dk/new-member/contents',
-              name: 'Web Onboarding DK - Contents',
-            }
           case 'dk-en':
-            return {
-              baseUrl: '/dk-en/new-member/contents',
-              name: 'Web Onboarding DK - Contents',
+            switch (match.params.name) {
+              case 'home':
+                return {
+                  baseUrl: '/dk/new-member/home',
+                  name: 'Web Onboarding DK - Contents',
+                }
+              case 'home-accident':
+                return {
+                  baseUrl: '/dk/new-member/home-accident',
+                  name: 'Web Onboarding DK - Danish Contents-Accident',
+                }
+              case 'home-accident-travel':
+                return {
+                  baseUrl: '/dk/new-member/home-accident-travel',
+                  name: 'Web Onboarding DK - Danish Contents-Accident-Travel',
+                }
```

Comment:
No reason to block this PR since we had a discussion on this, I just wonder what we will write on the landing page (haven't seen translations for that yet). Home, Home contents or just Contents? So we align what we call it with the url and the first 2 options would work, it's rather if we would call it Contents like in Norway it will be weird.

Reply:
It's Home Contents 🎉